ddp-workspace/projects/ddp-dsm-ui/src/app/FON/pipes/cutString.pipe.ts

```diff
@@ -0,0 +1,11 @@
+import {Pipe, PipeTransform} from '@angular/core';
+
+@Pipe({
+  name: 'cutString'
+})
+
+export class CutStringPipe implements PipeTransform {
+  transform(value: string, from: number = 17): string {
```

Comment:
it makes sense to use a trail as an input param, like
`transform(value: string, limit = 17, trail = '...') {
...
}`

Reply:
That's better. Thanks! Done!